GenumKit/Parsers/ColorsFileParser.swift

```diff
@@ -187,7 +187,7 @@ public final class ColorsXMLFileParser: ColorsFileParser {
                       namespaceURI: String?, qualifiedName qName: String?) {
       guard elementName == ColorsXMLFileParser.colorTagName else { return }
       guard let colorName = currentColorName, let colorValue = currentColorValue else { return }
-      parsedColors[colorName] = try? parse(hex: colorValue) ?? UInt32(0)
+      parsedColors[colorName] = try? parse(hex: colorValue)
```

Comment:
Do we want `parsedColors` dictionary to contain a `nil` in case the parsing is invalid (like `colorValue` is `foobar`), or should it contain `0`?

If the latter (use `0`), we should still coalesce using `??`, but the on the whole `try?` result:
```
parsedColors[colorName] = (try? parse(hex: colorValue)) ?? UInt32(0)
```
This way if the parse fails and throw, we store 0.
Or that could be a good occasion to actually print a proper error instead (= use `do/try/catch` and not `try?`)